apps/test/unit/lib/script-editor/StageDescriptionsTest.js

```diff
@@ -37,13 +37,10 @@ describe('StageDescriptions', () => {
     );
     assert.equal(wrapper.state('collapsed'), true);
     assert.equal(wrapper.childAt(1).children().length, 1);
-    assert.equal(
-      wrapper
-        .childAt(1)
-        .childAt(0)
-        .type(),
-      'button'
-    );
+
+    const button = wrapper.childAt(1).childAt(0);
```

Comment:
It looks like you had to change `const button` to `let button` to fix the other test - would this test be impacted by that same problem as well?

Reply:
not in this case -- I only had to do it in the other cases so that I could refresh the reference to the button after the component was updated by the click.

Reply:
👍